### DISCUSSION TITLE
Starter Project with ng-clarity

### DISCUSSION THREAD

**Discussion by binnyg** [vmware-clarity/ng-clarity#685] [General]
I want to request a starter project with ng-clarity. We do a ton of proof-of-concept projects and we always run into version conflicts when we upgrade the Angular version or clarity. We tried ngx-rocket with clarity but always ran into issues. 

Is it possible to maintain a starter project with the latest clarity components and best practices? This is to get off the ground quickly and start building the functionality.

**Comment by kevinbuhmann** [comment]
Please see our StackBlitz templates: https://stackblitz.com/@clr-team
